content/cloud-docs/api-docs/index.mdx

```diff
@@ -321,7 +321,8 @@ $ curl \
 
 ## Rate Limiting
 
-You can make up to 30 requests per second to the API as an authenticated or unauthenticated request. If you reach the rate limit then your access will be throttled and an error response will be returned.
+You can make up to 30 requests per second to the API as an authenticated or unauthenticated request. If you reach the rate limit then your access will be throttled and an error response will be returned. A few endpoints
+have lower limits to prevent email spam attack vectors. If you receive a rate limited response, the limit will be reflected in the `x-ratelimit-limit` header once triggered.
```

Comment:
```suggestion
have lower limits to prevent certain spam and abuse scenarios. If you receive a rate limited response, the limit will be reflected in the `x-ratelimit-limit` header once triggered.
```